helm/hpcc/templates/eclagent.yaml

```diff
@@ -163,7 +163,7 @@ spec:
 {{- if .useChildProcesses }}
 {{- include "hpcc.addResources" (dict "me" .resources) | indent 8 }}
 {{- else if not $.Values.global.privileged }}
-{{- include "hpcc.addStubResources" . | indent 8 }}
+{{- include "hpcc.addStubResources" (dict "instances" .maxActive) | indent 8 }}
```

Comment:
What happens if maxActive is not defined?

Reply:
good point, I meant default to 1

Reply:
@ghalliday - max made maxActive properties required